integration-tests/src/test/resources/application-test.yaml

```
@@ -8,6 +8,12 @@ ai:
+        "ai.koog": debug
+        # "ai.koog.agents.core.agent.GraphAIAgent": off
         "com.example.app": trace
-        "ai.koog.agents.core.agent.GraphAIAgent": off
-        "ai.koog.agents": warn
\ No newline at end of file
+
+management:
+    otlp:
+        tracing:
+            export:
+                enabled: off
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)